src/core/fonts.js

```diff
@@ -2836,7 +2840,7 @@ var Font = (function FontClosure() {
           !glyph.matchesForCache(fontChar, unicode, accent, width, vmetric,
                                  operatorListId, isSpace)) {
```

Comment:
Don't you need to add `isInFont` here as well (and also update the comparison in `Glyph.prototype.matchesForCache`)?